apps/e2e/ag-grid-storybook-e2e/src/e2e/ag-grid-stories.component.cy.ts

```diff
@@ -14,26 +14,28 @@ describe(`ag-grid-storybook`, () => {
         cy.get('#ready')
           .should('exist')
           .end()
+
+          // Expect back to top button to be visible.
+          .get('.sky-back-to-top')
+          .should('exist')
+          .should('be.visible')
+          .end()
+
+          // Expect the validation message to be visible.
+          .get('.sky-overlay sky-popover-content .sky-popover-body')
+          .should('exist')
+          .should('be.visible')
+          .should('contain.text', 'Expected a number between 1 and 18.')
+          .end()
```

Comment:
The component has actions in `ngAfterViewInit` that scroll a grid to show back-to-top as well as activate a validation popover. These assertions verify those have happened.

Reply:
This comment would be excellent to have in the source code, too.